index.html

```diff
@@ -4116,91 +4076,298 @@ <h3>Parsing content</h3>
-    the need for user prompting.
-  </p>
-  <p>
-    All <a>expressed permission</a>s that are preserved beyond the current
-    browsing session MUST be revocable.
-  </p>
+  <section class="informative"> <h3>Threats</h3>
+    <p>
+      Potential threats for Web NFC are given below.
+    </p>
+    <table class="simple" border="1">
+      <tr>
+        <th><strong>Threat</strong></th>
+        <th><strong>Comments, possible mitigation</strong></th>
+      </tr>
+      <tr>
+        <td>
+          Web page using the Web NFC API collects user data without the
+          consent of the user.
+        </td>
+        <td>
+          The user SHOULD be able to be aware of what data can be shared using
+          NFC from the given web page.
+          Use permissions and user prompts for accessing personal data,
+          minimize user data exposed to NFC.
+        </td>
+      </tr>
+      <tr>
+        <td>Malicious web page overwrites tags without user consent.</td>
+        <td>
+          Make sure the write is additive to previous content.
+          Or, require permission and user prompt needed for writing tags.
+          Or, control what tags can be written by a given web page, for instance
+          a web page can write only a tag which can be connected to its origin.
+          Or, allow overwriting since tags not meant to be written can be
+          protected by making them read only.
+        </td>
+      </tr>
+      <tr>
+        <td>
+          Malicious web page writes sensitive user data to tags and peer
+          devices.
+        </td>
+        <td>
+          The user should be able to be aware of what data can be shared using
+          NFC from the given web page.
+          Use permission/user prompt for writing tags and peers.
+        </td>
+      </tr>
+      <tr>
+        <td>
+          Malicious tag may be involuntarily or voluntarily read by devices
+          and the data read may constitute an attack vector on the user agent.
+        </td>
+        <td>
+          This is a generic problem with all existing NFC tags.
+          The data is considered application specific.
+          Implementations need security hardening.
+          Involuntary touch is low probability due to short range and critical
+          angle for reading.
+          It’s easier to attack elsewhere (e.g. WiFi and Bluetooth).
+        </td>
+      </tr>
+      <tr>
+        <td>
+          Malicious smart poster tag attempts triggering an action on the
+          device which may be a threat.
+        </td>
+        <td>
+          Support for smart posters is not among the main use cases of Web
+          NFC; if and when supported, no automatic actions should be allowed,
+          the user must be made aware and given the ability to control what is
+          happening during the NFC communication.
+        </td>
+      </tr>
+      <tr>
+        <td>
+          Malicious Web NFC tag could send an arbitrary payload to a website
```

Comment:
I don't like send here... the web page itself reads and needs to be aware that data might need sanitation, especially if made non-inert

Reply:
Will remove this threat as it doesn't add specific enough information.